### ISSUE TITLE
Support for Naga Epic

### ISSUE THREAD

**Issue by TheOneTheyCall-Tim** [1kc/razer-macos#855]
Love what you are doing here. I was pleased to see a bunch of Nagas supported. However, the Epic was not on the list. Sure enough, after installation, my Naga Epic was not recognized. I don't care so much about the lighting, but I was really hoping you could solve my inability to adjust the dpi sensitivity settings.

Thanks for doing what you do!
The One They Call... Tim

**Comment by TheOneTheyCall-Tim** [comment]
In further research looking at OpenRazer, I don't know enough to really know what I am looking at, but I see product IDs listed there for various products. Mine is not. 

Here is the info for the mouse I have: 

Razer Naga Epic:
  Product ID:	0x001f
  Vendor ID:	0x1532
  Version:	2.00
  Speed:	Up to 12 Mb/s
  Manufacturer:	Razer
  Location ID:	0x14113000 / 10
  Current Available (mA):	500
  Current Required (mA):	100
  Extra Operating Current (mA):	0

**Comment by stickoking** [comment]
I don't think this repo is being maintained anymore I have made a fork can you post you're issue there ? https://github.com/stickoking/razer-macos
